script/Ciel/Day 4/04_01_CIEL04_1.txt

```diff
@@ -177,87 +192,101 @@ It's not like I hate you or anything."
 [sha:5e5e2ce9526316d9b1933e0e135c58797872713c]{
 -- Page 17, Offset 8230.
 -- 「……いいです、言い訳なんかしなくても。
-"You don't have to make any excuses.
+"That's okay... you don't have to make excuses.
 }
 [sha:e04f78ebd87653b0221334561c942e3b12964b0c]{
 -- Page 17, Offset 8231.
 -- わたし、やっぱりお節介でしたよね……あと夕ご飯、コンビニ弁当は女子力低すぎでしたよね……」
-I knew I was meddling... Also, serving convenience store bentos for dinner isn't very feminine..."
+I overstepped my bounds, didn't I...
+ Plus, serving convenience store food for dinner doesn't reflect well on me as a girl..."
```

Comment:
japanese convenience store food is romantic af dw ciel

Reply:
<img width="46" alt="image" src="https://user-images.githubusercontent.com/16092757/180661169-8e6f4c9b-85ec-4c9d-bd39-ecd0a9e4a062.png">

Reply:
me and who
![IMG_20200101_193955_MP](https://user-images.githubusercontent.com/2476799/180662875-35fc0f7b-786a-4ba7-82b0-4b9c5dbd11a1.jpg)